website/pages/ar/deploying/deploying-a-subgraph-to-hosted.mdx

```diff
@@ -284,8 +284,13 @@ Developers will be notified by email if one of their subgraphs has been marked a
 
 ## Subgraph Studio subgraph archive policy
 
-When a new version of a subgraph is deployed, the previous version is archived (deleted from the graph-node DB). This only happens if the previous version is not published to The Graph's decentralized network.
+A Subgraph version in Studio is archived if and only if it meets the following criteria:
 
-When a subgraph version isn’t queried for over 45 days, that version is archived.
+- The version is not published to the network (or pending publish)
+- The version is not indexing a "special case" chain (polygon-zkevm, zksync-era, base, etc.)
+- The version was created 45 or more days ago
+- The subgraph hasn't been queried in 30 days
+
+In addition, when a new version is deployed, the N-2 version of the Subgraph is archived if it has not been published
```

Comment:
```suggestion
In addition, when a new version is deployed, if the subgraph has not been published, then the N-2 version of the subgraph is archived. 
```

Reply:
If my edit doesn't communicate the intended message, then feel free to disregard my suggestion; however, I attempted to make the conditional statement more clear. 
It's typically good practice to use conditional statements, such as "if, then" first. 
It's a user-friendly tech writing practice :)

Reply:
thanks for this, i did commit your suggestion as well!